packages/lib-react-components/src/MovableModal/MovableModal.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Rnd } from 'react-rnd'
+import ResizeIcon from './components/ResizeIcon'
+import withLayer from '../helpers/withLayer'
+import { Modal } from '../Modal'
+
+function MovableModal (props) {
+  const {
+    children,
+    closeFn,
+    headingBackground,
+    pad,
+    rndProps: {
+      minHeight,
+      minWidth,
+      position,
+      ...restRndProps
+    },
+    title,
+    ...rest
+  } = props
+
+  return (
+    <Rnd
+      cancel='.subtaskpopup-element-that-ignores-drag-actions'
+      enableResizing={{
+        bottom: true,
+        bottomLeft: false,
+        bottomRight: true,
+        left: true,
+        right: true,
+        top: true,
+        topLeft: false,
+        topRight: false
+      }}
+      default={position}
+      minHeight={minHeight}
```

Comment:
I think this may need to be adjusted a bit to ensure the `Modal` doesn't become too small. However, it could just be how to storybook is rendering this.

Reply:
The inner elements/child nodes have to have CSS for them to resize. I'll update the readme to make sure devs know this.